decidim-meetings/app/assets/javascripts/decidim/meetings/meetings_form_filter.component.js.es6

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-div-regex, no-useless-escape, no-param-reassign */
+((exports) => {
+  class MeetingsFormFilterComponent {
```

Comment:
Docs? I used jsdoc to document my components. Take a look at decidim-comments components.